app/src/androidTest/java/org/kiwix/kiwixmobile/utils/files/FileUtilsTest.kt

```diff
@@ -90,42 +119,42 @@ class FileUtilsTest {
   }
 
   @Test
-  fun `test decode file name`() {
```

Comment:
Don't create one fonction per string to test. Create one test with a array (input string, output string). Please test the specail cases with all the URL encode values.

Reply:
@kelson42 , done